client/src/components/Tabs/README.md

```diff
@@ -33,3 +33,88 @@ clickable tab.
  * `tabClassName` (string): Class to use for the tab.
 
  _NOTE:_ For other properties, please refer to the [reactstrap Tabs](https://reactstrap.github.io/components/tabs/) documentation.
+
+# Interacting with Tabs from a child component
+
+Components inside a tab can retrieve information about their parent using these method:
+- `useTabContext` which is a hook returning the tab context for functional components
+- `injectTabContext` which injects the tab context as props in your class component
+- `useTabFirstShow` which is a hook that allows you to execute a callback when a tab is first shown.
+
+The `tabContext` returned by `useTabContext` and `injectTabContext` contains the following properties
+- `activeTab` (string): name of the active tab in the nearest tab set
+- `currentTab` (string): name of the tab the current component resides in
+- `isOnActiveTab` (boolean): whether the current component is in a tab that is currently visible.
+
+The `tabContext` will be `false` for components outside of a tab.
+ 
+
+## Example of `useTabContext`
+
+```jsx
+import React from 'react';
+import useTabContext from 'hooks/useTabContext';
+
+export function MyComponent() {
+  // useTabContext only works inside functinal components
+  const tabContext = useTabContext();
+  if (!tabContext) {
+    return <p>I'm not in a tab.</p>;
+  }
+
+  const {activeTab, currentTab, isOnActiveTab} = tabContext;
+  
+  return (
+    <p>
+        The {activeTab} tab is currently active.
+        I'm on the {currentTab} tab which is {isOnActiveTab ? 'visible' : 'invisible'}.
+    </p>
+  ); 
+}
+```
+
+## Example of `injectTabContext`
+
+```jsx
+import React, {Component} from 'react';
+import {injectTabContext} from 'hooks/useTabContext';
+
+class MyComponent extends Component {
+  render({tabContext}) {  
```

Comment:
I'm not sure the `render({tabContext}) {` is correct

It threw errors on my local

However this worked as expected

```
render() {
  this.props.tabContext;
```

Reply:
You're right ... I've been doing too much functional components.